pydsa/dfs.py

```diff
@@ -0,0 +1,16 @@
+# Depth First Search
+# Complexity: O(V+E)
+#uses concept of list
+
+def dfsearch(graph,s):
```

Comment:
* `dfsearch` --> `dfs`
* `s` --> `start` Explicit is better than implicit (See [The Zen of Python](https://www.python.org/dev/peps/pep-0020/))
* space after comma (Follow [PEP8](https://www.python.org/dev/peps/pep-0008/) Standards)